src/main/java/com/thevoxelbox/voxelsniper/brush/Brush.java

```diff
@@ -200,6 +194,18 @@ protected final World getWorld() {
         return targetBlock.getWorld();
     }
 
+    protected final int getMinHeight() {
+        return getWorld().getMinHeight();
+    }
+
+    protected final int getMaxHeight() {
+        return getWorld().getMaxHeight();
+    }
+
+    protected final int clampWorldHeight(int height) {
```

Comment:
Why should this be a separate method? As far as I can see it is only used once? (not saying you should change it, just critical).

Reply:
For now that's true, but certain brushes might want to use it. And more methods is only better in my opinion. Improves readibility